src/viam/sdk/tests/test_common.cpp

```diff
@@ -115,6 +116,16 @@ BOOST_AUTO_TEST_CASE(test_from_dm_from_extra) {
     BOOST_CHECK_EQUAL(from_dm_from_extra(map), false);
 }
 
+BOOST_AUTO_TEST_CASE(test_version_metadata) {
+    // we don't want to check the specific values because they're going to be changing,
```

Comment:
How about we get the `sdk_version()` string here, and then check that it matches 
```
std::to_string(sdk_major_version()) + "." + std::to_string(sdk_minor_version()) + "." + std::to_string(sdk_patch_version());
```